server/src/main/java/thesistrack/ls1/constants/ThesisState.java

```diff
@@ -6,13 +6,13 @@
 @AllArgsConstructor
 @Getter
 public enum ThesisState {
-    PROPOSAL("Proposal"),
-    WRITING("Writing"),
-    SUBMITTED("Submitted"),
-    ASSESSED("Assessed"),
-    GRADED("Graded"),
-    FINISHED("Finished"),
-    DROPPED_OUT("Dropped out");
+    PROPOSAL("proposal"),
+    WRITING("writing"),
+    SUBMITTED("submitted"),
+    ASSESSED("assessed"),
+    GRADED("graded"),
+    FINISHED("finished"),
+    DROPPED_OUT("dropped_out");
```

Comment:
In `ApplicationState`, you use UPPER_CASEd names as a good practice. Let's do the same throughout all enums? 
```suggestion
    PROPOSAL("PROPOSAL"),
    WRITING("WRITING"),
    SUBMITTED("SUBMITTED"),
    ASSESSED("ASSESSED"),
    GRADED("GRADED"),
    FINISHED("FINISHED"),
    DROPPED_OUT("DROPPED_OUT");
```